### PR TITLE
Add vde-2 submodule for install

### PR DESCRIPTION
**Pull request by junnplus**
Signed-off-by: Ye Sijun <junnplus@gmail.com>

Integrated vde-2 submodule for easier installation.

### CODE REVIEW

**Comment by junnplus** [comment]
@jandubois Any suggestion?

File: README.md [hunk 1]
```diff
@@ -86,7 +71,7 @@ Use `/var/run/vde.bridged.en0.ctl` as the VDE socket path.
 ### Testing without launchd
 
 ```console
-make
+make install.vde-2
```
Comment (jandubois) [inline README.md]: Shouldn't this be `sudo make install.vde-2`?

And doesn't `make install.bin` require `sudo` as well?

**Review by jandubois — APPROVED** [review]
Thanks, LGTM, but please check the `sudo` question I commented on.

**Review by jandubois — APPROVED** [review]
Thanks!

**Pull request merged**